components/Button/Button.js

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export function Button({ series }) {
```

Comment:
Since you're making this a generic Button (or LinkButton) component it should not be tightly coupled to the idea of a series. Id suggest just expecting `{ href }` as props. Then construct the link url in the parent component, that actually cares about the concept of a series. This way you can reuse this wherever. 🥳 

```suggestion
export function LinkButton({ href }) {
```